versions/1.0.0/schema.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Conventional Changelog Configuration",
+    "description": "Describes the configuration options supported by conventional-config for upstream tooling.",
+    "type": "object",
+    "properties": {
+        "types": {
+            "description": "An array of `type` objects representing the explicitly supported commit message types, and whether they should show up in generated `CHANGELOG`s.",
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/type"
```

Comment:
We don't _have_ to use  `$ref` and `definitions` for the initial schema/module publication... this would just be the more JSON Schema-esque way to describe this object (for reuse).

It seems like forcing consumers to parse this as JSON Schema (following refs, other links) might be a little too much?

Reply:
I'm just not quite picturing how I'd appropriately populate the default types array based on this, is there a way we could describe the structure:

```
[
    { "type": "feat", "section": "Features"},
    { "type": "fix", "section": "Bug Fixes"},
    { "type": "test", "section": "Tests"},
    { "type": "build", "section": "Build System"},
    { "type": "ci", "hidden": true}
]
```

could we make the `default` a complex structure like this? otherwise I'm fine with the `$ref`.

Reply:
Ah, I can just add a `default` property under `properties.types.default` that would be the array (with it's contents) you just posted.

Reply:
Updated [here](https://github.com/conventional-changelog/conventional-changelog-config-spec/pull/8/files#diff-595a95901d174a00c8950a3f5d65b38fR13)... I'll add another test that validates the `default` values being exported against the schema itself.